src/test/java/org/mitre/synthea/engine/LogicTest.java

```diff
@@ -63,8 +64,9 @@ public void setup() throws IOException {
     person.attributes.put(Person.BIRTHDATE, 0L);
     time = System.currentTimeMillis();
     // Ensure Person's Payer is not null.
-    Payer.loadNoInsurance();
-    person.coverage.setPayerAtTime(time, Payer.noInsurance);
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```